mmv1/third_party/terraform/go.mod

```diff
@@ -32,8 +32,8 @@ require (
 	golang.org/x/exp v0.0.0-20240409090435-93d18d7e34b8
 	golang.org/x/net v0.30.0
 	golang.org/x/oauth2 v0.23.0
-	google.golang.org/api v0.203.0
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20241015192408-796eee8c2d53
+	google.golang.org/api v0.204.0
```

Comment:
In general we prefer to isolate dependency upgrades in their own PRs (https://googlecloudplatform.github.io/magic-modules/develop/update-dependencies/) - in this case since the tests are passing and the promotion is pretty simple, I'm tentatively okay with it.

Reply:
Thanks for the information. I will keep it in mind next time. Actually the intention was to merge the PR this week (I mean today. Since it could be release next monday in that scenario according to the release guideline) considering the urgency of the feature. Otherwise I would have wait for the version upgrade before promoting the feature. If you are ok with this (with simple change), it would be great!